x/execution/internal/keeper/keeper.go

```diff
@@ -300,16 +283,15 @@ func (k *Keeper) Update(ctx sdk.Context, msg types.MsgUpdate) (*executionpb.Exec
 		M.Completed.Add(1)
 	}
 
-	inst, err := k.instanceKeeper.Get(ctx, exec.InstanceHash)
-	if err != nil {
-		return nil, err
-	}
-	srv, err := k.serviceKeeper.Get(ctx, inst.ServiceHash)
-	if err != nil {
-		return nil, err
+	from := msg.Executor
```

Comment:
This is very weird:
Why the runner should be debited for the execution it just executed?

We don't know who created the execution, so we cannot debit it.
But, in the `create` function, there is this test:
```go
if proc == nil && run.Owner != msg.Signer.String() {
	return nil, sdkerrors.Wrap(sdkerrors.ErrUnauthorized, "signer is not the execution's executor")
}
```
Only the owner of the runner can create an execution is not attached to a process.
Thus, if the execution doesn't have a process, we could make it free. What do you think?

Reply:
Let's see that on another PR and totally remove the ability to create a standalone execution